ark/segmentation/marker_quantification.py

```diff
@@ -197,7 +202,10 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
         split_large_nuclei (bool):
             controls whether nuclei which have portions outside of the cell will get relabeled
         extraction (str):
-            extraction function used to compute marker counts.
+            extraction function used to compute marker counts
+        skip_extraction (bool):
+            if set, skips both the regionprops and the signal extraction steps regardless of
```

Comment:
This isn't true anymore. We should change the documentation, and also the name of the flag, to reflect the behavior. Maybe `fast_extraction`